src/main/resources/templates/movie_form.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
My understanding was that this tag was needed on each html page using thymeleaf formatting. If this works fine without then it clearly doesnt. <html xmls:th="www.thymeleaf.org">

Reply:
html xmls:th="www.thymeleaf.org" being the tag im referring to.